pytorch_lightning/trainer/trainer.py

```diff
@@ -204,6 +204,8 @@ def __init__(
 
             limit_test_batches: How much of test dataset to check (floats = percent, int = num_batches)
 
+            limit_predict_batches: How much of prediction dataset to check (floats = percent, int = num_batches)
```

Comment:
```suggestion
            limit_predict_batches: How much of prediction dataset to check (float = percent, int = num_batches)
```

Reply:
we need to say float = fraction, it can't be percent because the value is between 0 and 1

Reply:
Updated.